README.md

```diff
@@ -9,11 +9,11 @@ SPDX-License-Identifier: CC0-1.0
 
 NextFrame is a simple app based around the idea that actually, the Nextcloud topbar _is_ pretty cool after all.
 
-When installed, this app will create a new settings entry under `/settings/admin/additional` where you can generate a new "view token", which will be bound to a specific list of ancestors authorized by CSP.
+When installed, this app will create a new settings entry under `/settings/admin/additional` where you can generate a new "view token", which will be bound to a specific list of ancestors authorized by CSP. You should also configure reverse proxy, for nginx instructions see the [Configuring](./README.md#Configuring-nginx) section below.
 
 You can then embed the Nextcloud top bar onto another website, preferably on the same domain to guarantee shared login cookies.
 
-Here's an example of a recommended `iframe` tag making full use of Nextframe's features (requires [`iframeResizer.min.js`](https://raw.githubusercontent.com/davidjbradshaw/iframe-resizer/master/js/iframeResizer.min.js)):
+Here's an example of a recommended `iframe` tag making full use of Nextframe's features (requires [`iframeResizer.min.js`](https://github.com/davidjbradshaw/iframe-resizer/raw/dc27ef6028523950cfa422f403f94882634caba7/js/iframeResizer.min.js)):
```

Comment:
I don't think using a pinned commit is necessary.

Reply:
I written a pinned commit, because this file is missing in the `master` branch, and new files seems to be not compatible with NextFrame.

Reply:
You need to use the `js-dist/iframe-resizer.parent.js` in your parent file but thanks for the heads up